### DISCUSSION TITLE
Release 2.4.5

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#435] [General]
### Hello everyone! This update addresses a some bug fixes that have been done over the past week or two.

Changelog:
* PC Miner:
  * Fixed some reconnection issues
  * Changed socket send to sendall
  * Added periodic notice for powerful miners
  * Added threading lock to the print functions
  * Added new translations
  * Other small fixes and improvements

* AVR Miner & Code - **please upload the new Arduino Code to your board!**:
  * Fixed some reconnection issues
  * Changed socket send to sendall
  * Changed socket recv to select
  * Added threading lock to the print functions
  * Added new translations
  * Other small fixes and improvements
  * Optimized the Arduino Code (up to 15% higher hashrates!)
  * Improved stability

* ESP8266/ESP32 Codes:
  * Small improvements

### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

For the next release (2.5) we're planning to ship the pool system - it will be only for trusted people at first.
You can now also use the new [Duino-Coin Android app](https://play.google.com/store/apps/details?id=com.pripun.duinocoin) made by Pripun. 
In the meantime the master server was rewritten and should work nicely from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/2.4.5'>Release 2.4.5</a>.</em>

**Comment by Doofus4ever** [comment]
Just Curious how you going to determine who is trusted to test the pool system?

**Reply by revoxhere** [reply]
devs will be the first to test the system

**Reply by Doofus4ever** [reply]
oh ok fair enough lol cool beans
